### ISSUE TITLE
32-bit Windows installer from 0.5.37 failing on Win 7 Pro

### ISSUE THREAD

**Issue by edlins** [Ylianst/MeshCentral#1974]
MC: 0.5.37
Device OS: Windows 7 Pro 32-bit (plz don't laugh)

The installer fails silently, no `mesh` folder created in `C:\Program Files`.  When run in a cmd with `-fullinstall` we get the following:
```
C:\>meshagent32-AllCustomerSystems.exe -fullinstall                             
MeshCentral2 Agent                                                              
...Checking for previous installation [NONE]                                    
...Installing service [ERROR] fs.mkdirSync(): Unable to create dir: undefined\mesh
```
My guess is it's looking for an env var to find `C:\Program Files` but it's undefined.  If I'm right, I can define one and test again. 
 Another twist is, we were able to install these last year before we did an upgrade so it seems to have broken within the last year.

Thanks!

**Comment by krayon007** [comment]
v0.5.37 is about 6 months old. There was a bug I found with how I used IsWow64 on windows, that caused it to screw up the installation path. That fix was included in server version 0.5.51. 

I just verified that the installation works correctly with Windows 7 32 bit, with the current version of the agent, so if you try a more recent version of the server, it should resolve your problem.

**Comment by edlins** [comment]
Thanks!!  I'm reluctant to jump to the latest stable `0.5.81` because of the open issues regarding that version.  Jumping to `0.5.51` might be more agreeable to the team.  If I have problems I'll open a new issue.
